applications/matter_bridge/Kconfig

```diff
@@ -17,6 +17,10 @@ config BRIDGE_HUMIDITY_SENSOR_BRIDGED_DEVICE
 	bool "Support for Humidity Sensor bridged device"
 	default y
 
+config BRIDGE_GENERIC_SWITCH_BRIDGED_DEVICE
+	bool "Support for Generic Switch bridged device"
+	default y
```

Comment:
If the generic switch is not supported for simulated devices, we should add `depends on BRIDGED_DEVICE_BT`.